src/Jank/TermManagement/ui.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+
+const fontSize = '20';
```

Comment:
Not to sound nitpicky, but I think this should just be `20`, i.e. the number, not a string :grimacing: 
```suggestion
const fontSize = 20;
```

When used in a template string, this number will be converted to a string automatically. The reason for having it as a number outside the template string is also the reason to pull this out into its own variable - so we can use it outside the styling. For instance, if we later on want to implement a zoom feature for smaller and larger font-sizes, we can just multiply this. To make this work, this needs to be a number, however. Then we can do something like:
```
  font-size: ${fontSize * zoom}px;
```
where `zoom` is something like `1`, `1.5` or `2` based on the selected zoom level.

This is not a perfect example, a better one could be the usage of padding in the same way. Then you can calculate other attributes based on the padding, for instance:

```
const padding = 10;
const width = 100;
const Container = styled.div`
  padding: ${padding}px;
  width: ${width}px;
`
const Child = styled.div`
  width: ${width - (padding * 2)}px;
`;
```

If you prefer to keep it as a string after all, I suggest to also to have the unit in there, like
```suggestion
const fontSize = '20px';
```
so you don't have to repeat it everywhere - I'd vote for making it a number, though.